src/components/dashboard/dashboard.tsx

```diff
@@ -0,0 +1,363 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import BeatLoader from 'react-spinners/BeatLoader';
+import { DASHBOARD_CALLS, Get } from '../../common/dashboard';
+
+import '../analytic/analytic.scss';
+import './dashboard.scss';
+
+const Dashboard = () => {
+    const { t } = useTranslation();
+    const language = t('app.language');
+    const [userID, setUserID] = useState("cf9bfa6f-5837-42f9-9eeb-c15035c3c752");
+
+    //#region accouncements
+    const [announcementLoading, setAnnouncementLoading] = useState(true);
+    const [announcement, setAnnouncement] = useState("");
+
+    const getAnnouncement = () => {
+        // setErrLoading(false);
+        Get(DASHBOARD_CALLS.ANNOUNCEMENT,
+            //announcementGetTemp(
+            '',
+            { lang: language, userId: userID },
+            (responseData) => {
+                let item = responseData.data.Items ? responseData.data.Items[0] : {};
+                let announcement = "";
+                if (item)
+                    announcement = language === "fr" ? item.announcement_fr : item.announcement_en;
+                setAnnouncement(announcement);
+                setAnnouncementLoading(false);
+            },
+            (analyticErr) => {
+                // setErrMsg(errMsg.push(JSON.stringify(analyticErr)));
+                setAnnouncement("");
+                setAnnouncementLoading(false);
+            },
+            () => {
+                // setErrLoading(!nstLoading&&!nslLoading&&!nalLoading&&!natLoading&&!raaLoading&&!ralLoading&&!raoLoading&&!rslLoading);
+            }
+        );
+    };
+
+    useEffect(() => {
+        if (announcementLoading) {
+            getAnnouncement();
+        }
+    }, [announcementLoading, userID]);
+
+    useEffect(() => {
+        getAnnouncement();
+    }, [language]);
```

Comment:
when the component first time loaded, both language and userID changes will trigger the getAnnouncement called twice. (duplicate). language can be moved outside the component and passed from props, as a workaround.